### ISSUE TITLE
Talk on Data Products, R in Production, or R Package Development in Proprietary Settings

### ISSUE THREAD

**Issue by atheriel** [gta-r-user-group/gtar#6]
Hi, I'm Aaron Jacobs, a Data Scientist at Pinnacle. I work mostly on the production/software engineering aspect of our R ecosystem, so that's where I'd be most comfortable speaking. And I've never given a public talk, although I have given a few at work.

I have a couple different ideas, so here are some attempts at catchy titles for talks I'd like to give:

* **From Data Analysis to Data Products**. Most R users start out by doing solo data analysis and sometimes trying to build models. How do you move beyond "I did this thing on my computer" to a "data product" that can be used by team members or others in your organization? What do these paths look like in R? How do they change as the demands on this product grow?

* **R in Production: The Obvious Answer is the Right One**. R had a reputation for a long time as impossible to use in production. This is a bit weird in my view, because there are clear implementation paths for it, especially with modern microservice design patterns. I'll also share some of my own team's experience running R in production over the last year.

* **Developing R Packages in Proprietary Settings**. Most guides and tools for working with R packages seem to be geared at open source (and especially GitHub-oriented) models. I'll walk through what things look like when you are working on closed source packages inside a company or organization. Some of the differences are obvious, and some are not.

* **What Does R Value?** I wrote [a blog post](https://unconj.ca/blog/what-does-r-value.html) on this a few months ago that I'd love to turn into a talk. Basically, I think that the "platform values" of a language community play a significant role, and it can be illuminating to name them outright. What values does R hold above others? How can the history of the language inform our understanding of these values? And what does it mean for the kinds of things we can build with R?

Does any of that sound interesting?

**Comment by rich-iannone** [comment]
Great to meet you! All of these talk ideas sound great and I'm sure any one of them would be well-received at the Meetup. The hard part would be choosing one (my vote goes to the third one because I and maybe others in town could relate to the problem).

Would you like to talk timing? The April Meetup is booked but May would work.

**Comment by atheriel** [comment]
Glad you like them! I'm happy to give the third one, unless anyone else wants to chime in a vote for one of the others over the next little while.

And I could potentially do May. Is there a firm date already?

**Comment by rich-iannone** [comment]
Hi Aaron, sorry it took so long to get back to you on this. We have a date and venue lined up (June 4, 2019 at the Jonah Group). Would you be able to give a talk then? Thanks!

**Comment by atheriel** [comment]
Yep, I'm happy to give a talk then. After asking around, I think I'll go with my third pitch, "Developing R Packages in Proprietary Settings". Sound okay to you?

**Comment by rich-iannone** [comment]
I personally love this topic. Thanks! I’ll put the abstract in the event details.
